src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -2000,144 +1996,6 @@ public async Task<List<Tuple<OrganizationUser, string>>> RevokeUsersAsync(Guid o
         return result;
     }
 
-    public async Task RestoreUserAsync(OrganizationUser organizationUser, Guid? restoringUserId)
```

Comment:
Non-blocker: It's a bit difficult for me to distinguish between the existing code that was moved and the new code. Maybe in the future, we can separate them into different PRs to make code review easier.